ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -375,4 +374,14 @@ public void close() throws IOException {
         queuedIngestClient.close();
         streamingIngestClient.close();
     }
+
+    @Override
+    public void setQueueRequestOptions(RequestRetryOptions queueRequestOptions) {
+        queuedIngestClient.setQueueRequestOptions(queueRequestOptions);
+    }
+
+    @Override
+    public IngestionResourceManager getResourceManager() {
```

Comment:
Though I don't recall the description 100%, I think my comment is going to be related:
There's a bit of inconsistency with naming here, and I think we should get it right in this 1 PR. if it'll mess up the diff, maybe that can be the last step after PR approval?

1) If there's a class that implements an interface, the former should be more specific than the latter. Yet the naming implies the opposite: `class ResourceManager implements IngestionResourceManager`.
2) The type of this class' member is `IngestionResourceManager`, yet its name is `resourceManager`.
3) The name of this getter should align with both.

Reply:
I planned on changed the name after the PR ended to avoid having a huge diff.
Yochai suggested this name as a solution, but I still agree with you, I'll see what we can do.